contracts/axelar-gateway/src/contract.rs

```diff
@@ -170,18 +174,15 @@ impl AxelarGatewayInterface for AxelarGateway {
             .keccak256(&(CommandType::RotateSigners, signers.clone()).to_xdr(&env))
             .into();
 
-        let auth_module = Self::auth_module(&env);
-
         // TODO: Add rotation delay governance
         if env
             .storage()
             .persistent()
-            .has(&DataKey::RotationExecuted(data_hash.clone()))
-        {
+            .has(&DataKey::RotationExecuted(data_hash.clone())) {
             panic_with_error!(env, Error::RotationAlreadyExecuted);
         }
 
-        let is_latest_signers = auth_module.validate_proof(&data_hash, &proof);
+        let is_latest_signers = auth::validate_proof(&env, data_hash.clone(), proof.clone());
```

Comment:
```suggestion
        let is_latest_signers = auth::validate_proof(&env, data_hash.clone(), proof);
```
i don't think this `proof` needs to be cloned since it's not used for the rest of this function

Reply:
that's true. updated.